src/Storage/Helpers/InstanceHelper.cs

```diff
@@ -279,5 +279,27 @@ public static (string InstanceOwnerIdType, string InstanceOwnerIdValue) GetIdent
 
             return (string.Empty, string.Empty);
         }
+
+        /// <summary>
+        /// Validate and separate person number and organisation number
+        /// </summary>
+        /// <param name="InstanceOwnerIdType">The type of instance owner ID</param>
+        /// <param name="InstanceOwnerIdValue">The value of instance owner</param>
+        public static (string Person, string Org) ValidateSsnAndOrgNo(string InstanceOwnerIdType, string InstanceOwnerIdValue)
```

Comment:
I don't see any actual validation in this code, should the name be updated?

Reply:
How about `SeparatePersonAndOrgNo` ? 🤔